sys/net/link_layer/lorawan/Kconfig

```diff
@@ -0,0 +1,476 @@
+# Copyright (c) 2020 Freie Universitaet Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_USEMODULE_LORAWAN
+    bool "Configure LoRaWAN MAC"
+    depends on USEMODULE_LORAWAN
+    help
+        Configure LoRaWAN module using Kconfig. This may override configurations
+        in LoRa PHY.
+
+if KCONFIG_USEMODULE_LORAWAN
+
+choice
+    bool "LoRaWAN region"
+    default LORAMAC_REGION_EU_868
+    help
+        Configure the LoRaWAN region.
+
+config LORAMAC_REGION_EU_868
+    bool "EU863-870"
+
+config LORAMAC_REGION_IN_865
+    bool "IN865-867"
+
```

Comment:
Does the current port of the Semtech packet also support IN865-867? Does it support others? Perhaps it's  necessary to only show some regions depending on the stack that is used.

Reply:
This symbol is used to determine the channel plan in [region.h](https://github.com/RIOT-OS/RIOT/blob/master/sys/include/net/gnrc/lorawan/region.h#L31-L41). As of now it is limited to `GNRC_LORAWAN` stack only.

Reply:
Then it should only be available when GNRC LoRaWAN stack is used

Reply:
It seems that currently multiple regions are supported with the semtech package:
https://github.com/RIOT-OS/RIOT/blob/665c07eb535ba54562877af79e3862d9e717b583/tests/pkg_semtech-loramac/Makefile#L12

This is not limited to GNRC LoRaWAN it seems.

Reply:
Fixup made to allow for more regions in Semtech Stack.